taxonomies/categories.txt

```diff
@@ -80295,6 +80985,7 @@ fr:Pleurotes, Champignons pleurotes
 ciqual_food_code:en:20114
 ciqual_food_name:en:Oyster mushroom, raw
 ciqual_food_name:fr:Champignon, pleurote, crue
+# Is this a duplicate?
```

Comment:
Does not look like. Why?

```suggestion
```

Reply:
en:Oyster mushrooms appear again at line 81012.

Reply:
Indeed:
```
<en:Mushrooms
en:Oyster mushrooms, Pleurotus ostreatus
es:Setas de ostra, Setas ostra, Champiñones ostra, Gírgolas, Pleurotus ostreatus
fr:Pleurotes en huître, Pleurotus ostreatus
la:Pleurotus ostreatus
nl:Oesterzwammen
```

and 

```
<en:Mushrooms
en:oyster mushrooms
fr:Pleurotes, Champignons pleurotes
ciqual_food_code:en:20114
ciqual_food_name:en:Oyster mushroom, raw
ciqual_food_name:fr:Champignon, pleurote, crue
```

That can be merged too

Reply:
Okay, [done](https://github.com/Naruyoko/openfoodfacts-server/commit/090515a08780b2f1965237d8c98fabc59334739c).